src/main/java/org/mdbenefits/app/data/enums/MoneyOnHandType.java

```diff
@@ -0,0 +1,23 @@
+package org.mdbenefits.app.data.enums;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public enum MoneyOnHandType {
+    CHECKING("checking", "moneyonhand-types.checking-account"),
+    SAVINGS("savings", "moneyonhand-types.savings-account"),
+    BONDS("bonds", "moneyonhand-types.bonds"),
+    CASH_ON_HAND("cash", "moneyonhand-types.cash-on-hand"),
+    MUTUAL_FUNDS("mutual funds", "moneyonhand-types.mutual-funds"),
+    STOCKS("stocks", "moneyonhand-types.stocks");
+
+    private final String value;
```

Comment:
Will this `value` be the value for the PDF file?  You can use the `MoneyOnHandType.CHECKING.name()` for the value in the html.  Nat won't care that it's capitalized and we have a field that duplicates the name of the enum.  Though I could see having this field for the PDF value, or just using the labelSrc for that as well.

Reply:
Yes - the value here will be doing into the PDF file.